chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -31,6 +31,11 @@
 
   include_recipe "crowbar::stop-services-before-upgrade"
 
+  execute "delete pacemaker resources in non-DB cluster" do
+    command "cibadmin -E -f"
+    only_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
Maybe you should check for cibadmin, not crm?

Note that this will remove not just resources and constraints, but also nodes. But I guess that's okay...